README.md

```diff
@@ -125,7 +125,9 @@ repository.
 For more detailed usage of the package functionality, refer: [https://idealo.github.io/imagededup/](https://idealo.github.io/imagededup/)
 
 ## 👨‍💻 Streamlit Web App
-A simple streamlit based web application in order to find duplicates in a corpus of images using perceptual hashing, uploaded by the user. You can find the entire app's codebase in the ```streamlit_webapp``` folder :
+A simple streamlit based web application in order to find duplicates in a corpus of images using perceptual hashing, uploaded by the user. 
+
+GitHub Repo : https://github.com/prateekralhan/Imagededup---Streamlit
```

Comment:
I just checked the link. Will you add it there @prateekralhan 🙈

Reply:
@datitran , yes. I am yet to update my github repo with the necessary directories and files for the entire app's codebase. Just give me sometime please, I will update it and let you folks know so that the PR is good enough to be merged.

Reply:
@datitran I would like to review the new repo before we add the link in our Readme. 
@prateekralhan Could you please add the changes in your repo as a PR that I can review? (instead of adding all the remaining changes directly to the main branch. I see that right now you have only readme and license in there. So, maybe add me as a collaborator and allow me to review the changes you add in a new branch).

Reply:
@datitran and @tanujjain  I have updated the github repo's README. You can check it [here.](https://github.com/prateekralhan/Imagededup---Streamlit)

@tanujjain , Apologies I just now checked the PR conversation here and I have already made the repo README updates in main branch itself directly. Please have a look and let me know should any changes be needed.

Reply:
@tanujjain, Thank for the suggestions. I briefly went through all of them. Please give me some time to work on them and update the repo. Will keep you all posted on this. Thanks much!

Reply:
@tanujjain , I have implemented all the changes as per the suggestions you raised in the [issue.](https://github.com/prateekralhan/Streamlit-Imagededup/issues/1)

Github Repo: https://github.com/prateekralhan/Streamlit-Imagededup

@datitran  / @clennan , Happy to hear from you as well.

Reply:
Any updates? @tanujjain  / @datitran  / @clennan

Reply:
Hi, I need to find some time for this. Will let you know soon.

Reply:
@prateekralhan  So, I went through the codebase again. Unfortunately, the code quality is not up to our standards, so we will not be able to include the link to the repo in our readme. We appreciate the work you have put in and the output still remains impressive and useful, so we will be happy to share your work on our social media handles if you fix one issue - You're using `find_duplicates` method, but you should be using `find_duplicates_to_remove` method to get a unique list of duplicate files, otherwise, you will have repetition in the duplicate files list. (Read about the methods [here](https://idealo.github.io/imagededup/user_guide/finding_duplicates/)).

Please let us know if you're ok with this. If yes, please let us know when you have made the change and we will share the repo on our social media handles. Thanks for your understanding.

Reply:
@tanujjain , thank you for the feedback. I have made the changes and updated the repo (https://github.com/prateekralhan/Streamlit-Imagededup).
I am fine with the social media sharing thing. 😊

Reply:
@prateekralhan Thanks for the update.
@datitran This looks good to me. We can share the link on our socials.